packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2210,6 +2210,8 @@ This operation can only be performed on users with a `STAGED` or `DEPROVISIONED`
 
 Users who don't have a password must complete the welcome flow by visiting the activation link to complete the transition to `ACTIVE` status.
 
+> **Note:** If you want to send a branded User Activation email, change the subdomain of your request to the custom domain associated with the brand. For example, change `subdomain.okta.com` to `custom.domain.one`. See [Multibrand and custom domains](/docs/concepts/brands/#multibrand-and-custom-domains).
```

Comment:
Could consider "..domain associated..." to "...domain that's associated..." but it's ok as-is.

Reply:
Thanks, Tim. Updated.